compiler/luci-interpreter/src/BuddyMemoryManager.test.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci_interpreter/BuddyMemoryManager.h"
+#include <gtest/gtest.h>
+
+namespace luci_interpreter
+{
+namespace
+{
+
+using namespace testing;
+
+TEST(BuddyMemoryManager, BuddyMemoryManager)
+{
+  uint8_t *memPool = new uint8_t[200];
```

Comment:
1. Use lower case with underscore in local variable names
2. Use unique_ptr or simple array, current implementation will leak:
``` c++
  auto mem_pool = std::make_unique<uint8_t[]>(200);
```
``` c++
  uint8_t mem_pool[200];
```
P.s. first option is preferred if you want to allocate large memory pools.

Reply:
Done